openpype/hosts/houdini/api/lib.py

```diff
@@ -559,29 +559,51 @@ def get_template_from_value(key, value):
     return parm
 
 
-def get_frame_data(node):
+def get_frame_data(node, asset_data=None, log=None):
```

Comment:
I think this function would actually be clearer like this so that it's much clearer what might be going on with that data.

```suggestion
def get_frame_data(node, handle_start=0, handle_end=0, log=None):
```

Also note that the docstring `args` needs to be updated to include the new arguments.